pages/tournament/[tournament_id].tsx

```diff
@@ -44,22 +44,15 @@ export default function TournamentPage() {
           )}
           {isAdmin && <TournamentAdminPanel tournamentId={tournamentId} />}
           {tournamentQ.data ? (
-            <Questions
-              title={
-                tournamentQ.data?.name
-                  ? `${tournamentQ.data.name}`
-                  : "Loading..."
-              }
-              noQuestionsText="No questions in this tournament yet."
-              filterTournamentId={tournamentId}
-              description={tournamentQ.data?.description || undefined}
-            />
+            <>
+              <TournamentContent tournament={tournamentQ.data} />
+              <TournamentLeaderboard tournamentId={tournamentId} />
```

Comment:
Should we show the leaderboard in the carousel? I think mobile users will want to see that too. Should it go inside TournamentContent?

Reply:
Sure - I've moved this